DBN_article/DBN_article.md

```diff
@@ -0,0 +1,187 @@
+# Deep Belief Network
+
+&nbsp;&nbsp;&nbsp;In the modern landscape of innovation and technology, Artificial Intelligence (AI) invaded most 
+transformative and captivating advancements of our life. Starting with predictions on data, 
+classifying things into categories, and ending with pictures and music generation, AI is just 
+everywhere. One of the most outstanding application of AI is stuff creation (a.k.a. generation). 
+Today we will dive in discover one of the generative networks: Deep Belief Network (DBN).
+
+
+## Introduction
+
+&nbsp;&nbsp;&nbsp;Before we get into more specifics of DBNs, let’s go briefly from 
+some general concepts for a better understanding. It is common 
+practice to divide Machine Learning models into discriminative 
+and generative ones. As it might me concluded from the names, 
+discriminative models aim to separate data points into different 
+classes, while generative ones – to generate data points. 
+Generative models are usually used in unsupervised learning 
+problems as they are trained on inputs without their labels. 
+DBNs, that will be discussed today, are a type of deep learning 
+architecture that combine neural networks and unsupervised 
+learning.
+
+<div style="text-align: center">
+    <img src="dbn_1.jpg" alt="Fast_Learning" style="width:300px;"/>
+</div>
+
+## What are Deep Belief Networks (DBNs)?
+
+&nbsp;&nbsp;&nbsp;Deep Belief Network is a deep learning architecture, composed of 
+layers of Restricted Boltzmann Machines (RBMs), which are trained 
+in an unsupervised manner. \
+&nbsp; &nbsp; &nbsp;A Restricted Boltzmann Machines is a generative unsupervised 
+model used for feature selection and feature reduction technique, 
+for dimensionality reduction, classification, regression, and 
+other tasks in Machine Learning and Deep Learning. It is 
+learning on a probability distribution on a certain dataset 
+and used the learnt distribution to come up with conclusions an 
+unexplored data. A typical RBM architecture is represented below 
+(where h represents hidden nodes and v – visible nodes). \
+
+<div style="text-align: center">
+    <img src="dbn_2.png" alt="RBM architecture" style="width:400px;"/>
+</div>
+
+&nbsp; &nbsp; &nbsp;Coming back to DBNs, all RBMs it consists of, are trained in an 
```

Comment:
The phrase "all RBMs it consist of" confuses me a bit, could you elaborate around it?

Reply:
I understand the idea that DBNs are basically a lot of RBMs but in this formulation it created uncertainty for me